manifest/armv7l/g/git.filelist

```diff
@@ -8,176 +8,41 @@
-/usr/local/libexec/git-core/git-pack-redundant
-/usr/local/libexec/git-core/git-pack-refs
-/usr/local/libexec/git-core/git-patch-id
-/usr/local/libexec/git-core/git-prune
-/usr/local/libexec/git-core/git-prune-packed
-/usr/local/libexec/git-core/git-pull
-/usr/local/libexec/git-core/git-push
 /usr/local/libexec/git-core/git-quiltimport
-/usr/local/libexec/git-core/git-range-diff
-/usr/local/libexec/git-core/git-read-tree
-/usr/local/libexec/git-core/git-rebase
 /usr/local/libexec/git-core/git-receive-pack
-/usr/local/libexec/git-core/git-reflog
-/usr/local/libexec/git-core/git-refs
-/usr/local/libexec/git-core/git-remote
-/usr/local/libexec/git-core/git-remote-ext
-/usr/local/libexec/git-core/git-remote-fd
 /usr/local/libexec/git-core/git-remote-ftp
 /usr/local/libexec/git-core/git-remote-ftps
 /usr/local/libexec/git-core/git-remote-http
 /usr/local/libexec/git-core/git-remote-https
-/usr/local/libexec/git-core/git-repack
-/usr/local/libexec/git-core/git-replace
-/usr/local/libexec/git-core/git-replay
 /usr/local/libexec/git-core/git-request-pull
-/usr/local/libexec/git-core/git-rerere
-/usr/local/libexec/git-core/git-reset
-/usr/local/libexec/git-core/git-restore
-/usr/local/libexec/git-core/git-rev-list
-/usr/local/libexec/git-core/git-rev-parse
-/usr/local/libexec/git-core/git-revert
-/usr/local/libexec/git-core/git-rm
 /usr/local/libexec/git-core/git-send-email
-/usr/local/libexec/git-core/git-send-pack
 /usr/local/libexec/git-core/git-sh-i18n
 /usr/local/libexec/git-core/git-sh-i18n--envsubst
 /usr/local/libexec/git-core/git-sh-setup
 /usr/local/libexec/git-core/git-shell
-/usr/local/libexec/git-core/git-shortlog
-/usr/local/libexec/git-core/git-show
-/usr/local/libexec/git-core/git-show-branch
-/usr/local/libexec/git-core/git-show-index
-/usr/local/libexec/git-core/git-show-ref
-/usr/local/libexec/git-core/git-sparse-checkout
-/usr/local/libexec/git-core/git-stage
-/usr/local/libexec/git-core/git-stash
-/usr/local/libexec/git-core/git-status
-/usr/local/libexec/git-core/git-stripspace
 /usr/local/libexec/git-core/git-submodule
-/usr/local/libexec/git-core/git-submodule--helper
 /usr/local/libexec/git-core/git-svn
-/usr/local/libexec/git-core/git-switch
-/usr/local/libexec/git-core/git-symbolic-ref
-/usr/local/libexec/git-core/git-tag
-/usr/local/libexec/git-core/git-unpack-file
-/usr/local/libexec/git-core/git-unpack-objects
-/usr/local/libexec/git-core/git-update-index
-/usr/local/libexec/git-core/git-update-ref
-/usr/local/libexec/git-core/git-update-server-info
 /usr/local/libexec/git-core/git-upload-archive
 /usr/local/libexec/git-core/git-upload-pack
-/usr/local/libexec/git-core/git-var
-/usr/local/libexec/git-core/git-verify-commit
-/usr/local/libexec/git-core/git-verify-pack
-/usr/local/libexec/git-core/git-verify-tag
-/usr/local/libexec/git-core/git-version
 /usr/local/libexec/git-core/git-web--browse
-/usr/local/libexec/git-core/git-whatchanged
-/usr/local/libexec/git-core/git-worktree
-/usr/local/libexec/git-core/git-write-tree
```

Comment:
Do we want to remove all these git-core files?

Reply:
Those look like they are internal now. For instance, git bisect appears to work...